packages/ember-runtime/lib/system/native_array.js

```diff
@@ -114,13 +114,16 @@ NativeArray = NativeArray.without.apply(NativeArray, ignore);
 
   Example
 
+  TODO: Update example to not use CollectionView
```

Comment:
Need to kill this TODO comment, looks like the docs below already took care of it...